harvard-lts/Harvard-Geospatial-Library
# Update Dockerfile

**Remove vim tiny*
* * *

# What does this Pull Request do?
We don't want vim in the image

# How should this be tested?

Build this branch, exec into the container and make sure the vim/vi commands no longer work (re: try vim pizza.txt)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
